html-aam/index.html

```diff
@@ -13090,7 +13096,7 @@ <h4 id="att-optimum">`optimum`</h4>
             <tr>
               <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
               <td>
-                <div class="general">Not mapped</div>
+                <div class="general">Expose "optimal value" via `AXValueDescription` if &lt;value&gt; is greater than or equal to &lt;optimum&gt;.</div>
```

Comment:
[@rahimabdi 's migrated comment](https://github.com/w3c/html-aam/pull/538/files#r1585789869):

Relevant WebKit code: https://github.com/WebKit/WebKit/blob/c330ea99229f6bb478dde74497ee652e44a3d7d5/Source/WebCore/html/HTMLMeterElement.cpp#L184

For optimum:

If value is greater than or equal to low AND value is less than or equal to high then value is "optimal value"
Else, value is "suboptimal value"
Note that in this case, there is no "critical value".